packages/ingest/src/recursion.ts

```diff
@@ -2,12 +2,10 @@ import path from 'path';
 import * as log from '@cumulus/common/log';
 
 export interface RecursionFile {
-  name: string,
-  type: string
+  type: number,
```

Comment:
How could this type change so significantly?

Reply:
🤷‍♂️

Reply:
I'll rephrase: why did we have change it?

Reply:
Before this PR, nothing that was actually using `recursion.ts` had been migrated to TypeScript. This PR starts using that recursion code, and it flagged the fact that `type` is actually a number, not a string.